packages/next-frontend/src/app/[locale]/layout.tsx

```diff
@@ -8,6 +12,7 @@ import { getMessages } from "next-intl/server";
 import type { Locale } from "@rainbow-me/rainbowkit";
 import { ClientProviders } from "../../components/client-providers";
 import { Nav } from "../../components/nav";
+
 import styles from "./styles.module.css";
```

Comment:
Can we move this above alongside the other CSS imports?

Reply:
Fixed in 553b93b44f72fa608f10241ebc55277f6270e045.